### ISSUE TITLE
RC12 IE11 Polyfill error

### ISSUE THREAD

**Issue by sunnyyadav100** [SAP/ui5-webcomponents#2866]
**Describe the bug**

I am trying to update UI5 web-components version to RC12. Followed changed log https://github.com/SAP/ui5-webcomponents/releases/tag/v1.0.0-rc.12 and installed @ui5/webcomponents-ie11 package to support IE11.

During runtime getting below error - 
`Uncaught TypeError: Cannot read property 'observeChildren' of undefined`

My application is based on Angular 8.

**To reproduce**
2. ...
3. ...

**Isolated example**
Please provide an isolated example if possible, by forking [CodeSandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html)

**Expected behavior**

**Screenshots**

<img width="1518" alt="image" src="https://user-images.githubusercontent.com/9139037/109034765-fb471b00-76ed-11eb-84c0-9f50282088aa.png">

**Context**
- UI5 Web Components version 1.0.0-rc.12
- OS/Platform: {...}
- Browser *(if relevant)*: {...}
- Other information: {...}

**Affected components** *(if known)*

**Log output / Any errors** in the console
```
integrate.js:21 Uncaught TypeError: Cannot read property 'observeChildren' of undefined
    at Module.../../node_modules/@ui5/webcomponents-ie11/dist/integrate.js (integrate.js:21)
    at __webpack_require__ (bootstrap:84)
    at Module.../../node_modules/@ui5/webcomponents-ie11/dist/features/IE11.js (IE11.js:1)
    at __webpack_require__ (bootstrap:84)
    at Module.../../node_modules/@ui5/webcomponents-ie11/dist/features/IE11WithWebComponentsPolyfill.js (IE11WithWebComponentsPolyfill.js:1)
    at __webpack_require__ (bootstrap:84)
    at Module../src/polyfills.ts (polyfills.ts:1)
    at __webpack_require__ (bootstrap:84)
    at Object.2 (polyfills.ts:91)
    at __webpack_require__ (bootstrap:84)
../../node_modules/@ui5/webcomponents-ie11/dist/integrate.js @ integrate.js:21
__webpack_require__ @ bootstrap:84
../../node_modules/@ui5/webcomponents-ie11/dist/features/IE11.js @ IE11.js:1
__webpack_require__ @ bootstrap:84
../../node_modules/@ui5/webcomponents-ie11/dist/features/IE11WithWebComponentsPolyfill.js @ IE11WithWebComponentsPolyfill.js:1
__webpack_require__ @ bootstrap:84
./src/polyfills.ts @ polyfills.ts:1
__webpack_require__ @ bootstrap:84
2 @ polyfills.ts:91
__webpack_require__ @ bootstrap:84
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ polyfills.js:1
compiler.js:2420 Uncaught Error: Can't resolve all parameters for ApplicationModule: (?).
    at syntaxError (compiler.js:2420)
    at CompileMetadataResolver.push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver._getDependenciesMetadata (compiler.js:21576)
    at CompileMetadataResolver.push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver._getTypeMetadata (compiler.js:21468)
    at CompileMetadataResolver.push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver.getNgModuleMetadata (compiler.js:21336)
    at CompileMetadataResolver.push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver.getNgModuleSummary (compiler.js:21146)
    at compiler.js:21260
    at Array.forEach (<anonymous>)
    at CompileMetadataResolver.push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver.getNgModuleMetadata (compiler.js:21248)
    at CompileMetadataResolver.push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver.getNgModuleSummary (compiler.js:21146)
    at compiler.js:21233
syntaxError @ compiler.js:2420
push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver._getDependenciesMetadata @ compiler.js:21576
push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver._getTypeMetadata @ compiler.js:21468
push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver.getNgModuleMetadata @ compiler.js:21336
push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver.getNgModuleSummary @ compiler.js:21146
(anonymous) @ compiler.js:21260
push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver.getNgModuleMetadata @ compiler.js:21248
push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver.getNgModuleSummary @ compiler.js:21146
(anonymous) @ compiler.js:21233
push.../../node_modules/@angular/compiler/fesm5/compiler.js.CompileMetadataResolver.getNgModuleMetadata @ compiler.js:21211
push.../../node_modules/@angular/compiler/fesm5/compiler.js.JitCompiler._loadModules @ compiler.js:27378
push.../../node_modules/@angular/compiler/fesm5/compiler.js.JitCompiler._compileModuleAndComponents @ compiler.js:27359
push.../../node_modules/@angular/compiler/fesm5/compiler.js.JitCompiler.compileModuleAsync @ compiler.js:27319
push.../../node_modules/@angular/platform-browser-dynamic/fesm5/platform-browser-dynamic.js.CompilerImpl.compileModuleAsync @ platform-browser-dynamic.js:143
compileNgModuleFactory__PRE_R3__ @ core.js:26576
push.../../node_modules/@angular/core/fesm5/core.js.PlatformRef.bootstrapModule @ core.js:26790
./src/main.ts @ main.ts:11
__webpack_require__ @ bootstrap:84
0 @ state-reducer.ts:19
__webpack_require__ @ bootstrap:84
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ main.js:1
zone.js:703 Unhandled Promise rejection: Cannot read property 'cssVars' of undefined ; Zone: <root> ; Task: Promise.then ; Value: TypeError: Cannot read property 'cssVars' of undefined
    at EventProvider.runPonyfill (CSSVarsPonyfill.js:26)
    at EventProvider.js:53
    at Array.map (<anonymous>)
    at EventProvider.fireEvent (EventProvider.js:52)
    at fireThemeLoaded (ThemeLoaded.js:15)
    at _callee4$ (applyTheme.js:75)
    at tryCatch (runtime.js:62)
    at Generator.invoke [as _invoke] (runtime.js:288)
    at Generator.prototype.<computed> [as next] (runtime.js:114)
    at asyncGeneratorStep (ThemeLoaded.js:14) TypeError: Cannot read property 'cssVars' of undefined
    at EventProvider.runPonyfill (http://localhost:4200/polyfills.js:5573:26)
    at http://localhost:4200/polyfills.js:3405:34
    at Array.map (<anonymous>)
    at EventProvider.fireEvent (http://localhost:4200/polyfills.js:3404:29)
    at fireThemeLoaded (http://localhost:4200/polyfills.js:4642:24)
    at _callee4$ (http://localhost:4200/polyfills.js:4832:84)
    at tryCatch (http://localhost:4200/polyfills.js:13163:17)
    at Generator.invoke [as _invoke] (http://localhost:4200/polyfills.js:13382:22)
    at Generator.prototype.<computed> [as next] (http://localhost:4200/polyfills.js:13216:21)
    at asyncGeneratorStep (http://localhost:4200/polyfills.js:4663:103)
```

**Organization:** *(if applicable)*
SAP Ariba

**Priority:** *(optional)* *(Low, Medium, High or Very High)*
Medium

Setting priority would indicate the importance of the issue
and would help the process of dispatching and processing.

To set the appropriate priority, you need to consider the impact
and if the issue:
- Breaks entire application or system - High and Very High
- Is accessibility one (Screan Reader, Keyboard Handling) - Medium and High
- Is functional one and makes a single component unusable - Medium and High
- Is visual, styling one - Low and Medium

Note: the priority would be re-evaluated by the issue processors, according to the project internal criterias

**Business impact:** *(mandatory for "Very High")*

**Comment by ilhan007** [comment]
Hello @sunnyyadav100 can you also provide us a link to the repo or to the deployed app.

BR,
ihan

**Comment by fifoosid** [comment]
Hi @sunnyyadav100 

This issue is resolved in master branch by the following PR: #2878

It is released in our @ next version ```0.0.0-5ffbbd9fe``` available in NPM & it will be released in rc13 as well.

**Comment by sunnyyadav100** [comment]
Thanks
